### DISCUSSION TITLE
Disable Sing Up?

### DISCUSSION THREAD

**Discussion by leoborlot** [usememos/memos#505] [General]
I want to use this app for my team, and dont want more people submiting accounts. How May I disable the singup optin in menu and close registra page?

**Comment by boojack** [comment]
There is an "allow user signup" option in setting -> system. You can set it false to disable signup.

<img width="896" alt="image" src="https://user-images.githubusercontent.com/24653555/202931139-0c9e6a6f-0b3d-4d85-b315-03c10fb2b0c7.png">

**Comment by leoborlot** [comment]
I found, thank you!

**Reply by IamTaoChen** [reply]
If I disable sign up. and a new user can not login with Oauth2, my memos version is 0.16
